prescription-sync-job/overlays/prod/cronjob.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
we can remove the `prod` overlays, we are no longer maintaining them.

Reply:
Removed in 339dcf4 :+1: